docs/guide/net.rst

```diff
@@ -70,19 +71,115 @@ This will return a list-like object which contains one or more filepaths to the
+The other is the `~dkist.io.FileManager.download` method of ``Dataset.files``, this method allows you to only download some FITS files based on the slicing of a dataset object.
+
+
+Downloading Files from a ``Dataset``
+####################################
+
+The most flexible way to download FITS files is first having loaded the dataset from an ASDF file.
+This workflow enables you to choose what FITS files you want to transfer based on the metadata of the dataset, for example, the wavelength or stokes profile.
+
+The first step in this workflow is to construct a `dkist.Dataset` object from an already downloaded ASDF.
+
+How to do this is detailed in the next section :ref:`loadinglevel1data`, but a quick example, following on from the ``Fido.fetch`` call above:
 
 .. code-block:: python
 
    ds = dkist.Dataset.from_asdf(files[0])
 
+Once the dataset is loaded, we can use the `dkist.Dataset.files` property to manage where the dataset looks for the FITS files associated with the dataset.
+By default the ``Dataset`` object will assume the FITS files are in the same directory as the ASDF file that was loaded.
+You can see what this directory is by using the ``ds.files.basepath`` property.
+
+.. code-block:: python
+
+   ds.files.basepath
+
+If you wish to re-point the dataset to look for the FITS files in another directory you can explicitly set this property.
+For example:
+
+.. code-block:: python
+
+   ds.files.basepath = "~/data/dkist/BCDEF"
+
+.. note::
+
+   To transfer files to your local computer you will need a running instance of the `Globus Connect Personal <https://www.globus.org/globus-connect-personal>`__ software.
+   All the following documentation assumes you have this running and wish to transfer files using GCP to the machine where your Python session is running.
+   It is possible to use the `dkist` package to orchestrate transfers to remote endpoints or other more complex arrangements by specifying the ``destination_endpoint=`` keyword argument to all these functions.
+
 Once we have loaded the dataset, if we wish to transfer all the FITS files a single call to `~dkist.io.FileManager.download` will initiate the transfer:
 
 .. code-block:: python
 
    ds.files.download()
 
-If this is the first time you have run this method a login page should open in your webbrowser to authenticate with the Globus system.
-Without overriding any of the default arguments calling this method **expects you to already have a running Globus personal endpoint** local to the Python session.
-If you wish to transfer the dataset to a non-local endpoint, you can do so by setting the ``destination_endpoint=`` keyword argument.
+If this is the first time you have run this method, or your authentication has expired, a login page should open in your webbrowser to authenticate with the Globus system.
+By default this call will download all the FITS files to the current value of ``ds.files.basepath``, i.e. next to the loaded ASDF file.
```

Comment:
"i.e., next to the loaded ASDF file.": Based on lines 105 above my read is that ``ds.files.basepath`` isn't _always_ next to the ASDF file. Is that right? If so maybe text like "...basepath`` (defualts to the same directory as the loaded ASDF file)." would be more correct.